pinot-common/src/main/java/org/apache/pinot/common/metrics/AbstractMetrics.java

```diff
@@ -58,6 +57,9 @@
 
   private final Class _clazz;
 
+  // The purpose of having _gaugeValues is to make gauge metric updatable.
+  // Since gauge metric itself is updatable now, we can deprecate it.
```

Comment:
Should we add the PR that made this change?

Reply:
added.